tensorflow/core/common_runtime/bfc_allocator.cc

```diff
@@ -178,7 +178,8 @@ bool BFCAllocator::Extend(size_t alignment, size_t rounded_bytes) {
 }
 
 BFCAllocator::ChunkHandle BFCAllocator::AllocateChunk() {
-  if (free_chunks_list_ != kInvalidChunkHandle) {
+  const int kInvalidChunkHandle_int = kInvalidChunkHandle;
+  if (free_chunks_list_ != kInvalidChunkHandle_int) {
```

Comment:
Why don't we change type of `kInvalidChunkHandle` where it is defined?

Reply:
Where is it defined?

Reply:
https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/core/common_runtime/bfc_allocator.h;drc=59283c98c9abdedd34657d326cec9ce10460c987;l=136

Reply:
kInvalidChunkHandle is apparently a `-1` constant, while the type it's being compared to `ChunkHandle` is size_t. 
->
`KInvalidChunkHandle` cannot be cast to size_t without necessarily losing its intended functionality.

In these cases it should actually be `free_chunks_list_ ` /  h`  that should be cast. I'll make the update.